.github/workflows/ci.yml

```diff
@@ -5,10 +5,22 @@ on:
     branches:
       - dev
       - master
+    paths-ignore:
+      - 'README*.md'
```

Comment:
Would it be a good idea to ignore all *.md files?

Reply:
Maybe, but I tried to be conservative here so so that there are no false negatives.

Reply:
Hmm, I still don't feel glad about picking so many paths explicitly.

I don't think there can be any false positives (at least currently):

All ``.md`` files are either ``Readme.md``'s or stuff in the ``.github`` folder:
https://github.com/TeamNewPipe/NewPipe/search?l=Markdown&p=1

So maybe just go with the simpler ``**.md``?
Or what about simply ignoring all ``.md``'s in the ``.github`` folder?

Reply:
My concern is that somebody might change something in the code to depend on one of these files (or adds one), and then they forgot to change the paths listed here. However, I'm not strongly opposed to being less conservative and ignoring more things.

Reply:
Okay.

However consider also the opposite situation could occur. 
e.g. someone adds a new ``.md`` to the ``.github`` folder and thinks it's ignore by the CI.

Anyway, I'm also fine with the current method.
But can you at least also add ``.github/CONTRIBUTING.md``?

Reply:
> someone adds a new .md to the .github folder and thinks it's ignore by the CI.

Not everybody is necessarily aware that files that the CI ignore actually exist, so it's more likely for a mistake to happen in this case I'd say. Anyway, I'm going to add `.github/**/*.md` then.